### DISCUSSION TITLE
Paste copy exception

### DISCUSSION THREAD

**Discussion by zonglin-hub** [helix-editor/helix#9938] [Troubleshooting]
![image](https://github.com/helix-editor/helix/assets/87891147/e349bc15-c0a4-4371-bc89-3c4340eae2d4)

![image](https://github.com/helix-editor/helix/assets/87891147/b13eddb1-c146-4635-a697-39d48142f66d)

https://github.com/helix-editor/helix/assets/87891147/edb5446f-5974-4927-a2d9-80aa987e02d7

**Comment by kirawi** [comment]
This isn't the right use of "exception" nor does Rust have exceptions. But see https://github.com/helix-editor/helix/discussions/9243#discussioncomment-8019541
